### DISCUSSION TITLE
0.0.9

### DISCUSSION THREAD

**Discussion by nhaouari** [nhaouari/obsidian-textgenerator-plugin#13] [Announcements]
This release of Text Generator Plugin V0.0.9 includes better error handling and support for ymal configuration (exprimental).

This will help make your experience with the plugin more professional and enjoyable.

<hr /><em>This discussion was created from the release <a href='https://github.com/nhaouari/obsidian-textgenerator-plugin/releases/tag/0.0.9'>0.0.9</a>.</em>

**Comment by ShravanSunder** [comment]
Would it be possible to add presence and other paramters?  https://beta.openai.com/docs/api-reference/completions/create#completions/create-temperature. 

Can we currently do it now with ymal?  And is it possible to use json as a configuration file?

**Reply by nhaouari** [reply]
yes, it is possible through YMAL configuration: [https://text-gen.com/frontmatter-configuration]( https://text-gen.com/frontmatter-configuration)
